website/pages/en/querying/graphql-api.mdx

```diff
@@ -49,6 +49,23 @@ When querying a collection, the `orderBy` parameter may be used to sort by a spe
 }
 ```
 
+#### Example for nested entity sorting
+
+Sorting on the basis of nested entities is useful if you are looking to sort entities based on child fields.
+
+In the following example, we sort the tokens by the name of their owner:
+
+```graphql
+{
+  tokens(orderBy: owner__name, orderDirection: asc) {
+    id
+    owner {
+      name
+    }
+  }
+}
+```
+
```

Comment:
Is it worth adding a note on constraints (e.g. only works one level deep, doesn't work for interfaces - anything else?)

Reply:
presumably it doesn't work if the nested entities are arrays (rather than single linked entities)?

Reply:
Presumably it _does_ work on derived entities? I think worth adding these kind of clarifiers